server/src/routes/api/v1/showsReviewsRouter.js

```diff
@@ -24,4 +28,33 @@ reviewsRouter.post("/", async (req,res) => {
   }  
 })
 
+reviewsRouter.delete("/:id", async (req, res) => {
+  try {
+    const reviewId = req.params.id
+    await Review.query().deleteById(reviewId)
+    return res.status(200).json()
+  } catch(error) {
+    return res.status(500).json({ error })
+  }
+})
+
+reviewsRouter.patch("/edit", async (req, res) => {
+  try {
+    const editedReview = cleanUserInput(req.body)
+    const reviewId = editedReview.reviewId
+    await Review.query().update({
+      body: editedReview.body,
+      score: editedReview.score,
+      showId: editedReview.showId,
+      userId: editedReview.userId })
+      .where('id', reviewId)
```

Comment:
Good attempt, however the chaining of where here is unecessary and actually costly. Prefer `updateAndFetchById`
https://vincit.github.io/objection.js/api/query-builder/mutate-methods.html#updateandfetchbyid